packages/thirdweb/src/wallets/smart/smart-wallet-integration-v07.test.ts

```diff
@@ -224,29 +224,29 @@ describe.runIf(process.env.TW_SECRET_KEY).sequential(
       let isDeployed = await isContractDeployed(newSmartAccountContract);
       expect(isDeployed).toEqual(false);
 
-      // sending transactions in parallel should deploy the account and not cause errors
-      const txs = await Promise.all([
-        sendAndConfirmTransaction({
-          transaction: claimTo({
-            contract,
-            quantity: 1n,
-            to: newSmartAccount.address,
-            tokenId: 0n,
-          }),
-          account: newSmartAccount,
+      const tx1 = await sendAndConfirmTransaction({
```

Comment:
Don't await these, that's what the test is for. Sending 2 txs in a row without waiting for confirmation.

Would still do the promise all, but the second promise wrapped in a 1s timeout so it fires off the tx slightly later

Reply:
ok